app/util/Acl.scala

```diff
@@ -47,4 +48,49 @@ class Acl(permissions: PermissionsProvider) extends Logging {
       testUser(permission, switch)(email)
     else AccessGranted
   }
+
+  def testUserGroupsAndCollections(editorialPermission: PermissionDefinition, commercialPermission: PermissionDefinition,
```

Comment:
I'd just reference the permissions directly rather than passing them as parameters, since the variable names already match up to specific ones

Reply:
We need to pass in the editorial permissions here because it can be one of two permissions (either just edit or edit and launch). We only have one kind of permission for commercial and training so they don't have to be passed in here but I think the symmetry makes it easier to read.